tests/integration/components/imgix-image-test.js

```diff
@@ -1,85 +1,126 @@
 import { moduleForComponent, test } from 'ember-qunit';
 import hbs from 'htmlbars-inline-precompile';
 
-/* global URI */
-
 moduleForComponent('imgix-image', 'Integration | Component | imgix image', {
   integration: true
 });
 
 test('it renders', function(assert) {
   this.render(hbs`{{imgix-image path="/users/1.png"}}`);
+  assert.ok(this.$());
+});
 
-  assert.equal(this.$().text().trim(), '');
-  assert.equal(this.$('img').length, 1);
+test('it renders event more better', function(assert) {
+  this.render(
+    hbs`<div style='width:200px;height:200px;'>{{imgix-image path='/users/1.png' }}</div>`
+  );
+
+  let url = new URL(this.$('img').attr('src'));
+  assert.equal(
+    this.$()
+      .text()
+      .trim(),
+    ''
+  );
+  assert.equal(url.pathname, '/users/1.png');
 });
 
 test('it renders the correct path', function(assert) {
   this.render(hbs`{{imgix-image path="/users/1.png"}}`);
 
-  assert.ok(this.$('img').attr('src').indexOf("https://assets.imgix.net/users/1.png") > -1);
-  assert.ok(this.$('img').attr('src').indexOf("w=1280") > -1);
+  assert.ok(
+    this.$('img')
+      .attr('src')
+      .indexOf('https://assets.imgix.net/users/1.png') > -1
+  );
+  assert.ok(
+    this.$('img')
+      .attr('src')
+      .indexOf('w=1250') > -1
+  );
 });
 
 test('it builds the default URL', function(assert) {
   this.render(hbs`{{imgix-image path="/users/1.png"}}`);
-  let url = URI(this.$('img').attr('src'));
+  let url = new URL(this.$('img').attr('src'));
 
-  assert.equal(url.search(true).w, "1280");
-  assert.equal(url.pathname(), "/users/1.png");
-  assert.equal(url.search(true).fit, "crop");
-  assert.equal(url.search(true).crop, "faces");
+  assert.equal(url.searchParams.get('w'), '1250');
+  assert.equal(url.pathname, '/users/1.png');
+  assert.equal(url.searchParams.get('fit'), 'crop');
+  assert.equal(url.searchParams.get('crop'), 'faces');
 });
 
 test('it maintains any query parameters passed in', function(assert) {
   assert.expect(2);
   this.render(hbs`{{imgix-image path="/users/1.png?sat=100"}}`);
 
-  let url = URI(this.$('img').attr('src'));
-  assert.equal(url.search(true).sat, "100");
-  assert.equal(url.search(true).w, "1280");
+  let url = new URL(this.$('img').attr('src'));
+  assert.equal(url.searchParams.get('sat'), '100');
+  assert.equal(url.searchParams.get('w'), '1250');
 });
 
 test('it renders with an aspect ratio', function(assert) {
   this.render(hbs`{{imgix-image path="/users/1.png" aspectRatio=1.3333}}`);
 
-  assert.equal(this.$().text().trim(), '');
-  let url = URI(this.$('img').attr('src'));
+  assert.equal(
+    this.$()
+      .text()
+      .trim(),
+    ''
+  );
+  let url = new URL(this.$('img').attr('src'));
 
-  assert.equal(url.search(true).w, "1280");
-  assert.equal(url.search(true).h, "960");
+  assert.equal(url.searchParams.get('w'), '1250');
```

Comment:
Same comment as above--relying on the window width here seems brittle. Wrapping this component in a specifically-sized container would be a lot more reliable.